glusterd2/transaction/context.go

```diff
@@ -139,69 +162,36 @@ func (c *Tctx) GetNodeResult(nodeID uuid.UUID, key string, value interface{}) er
 
 // Delete deletes the key and attached value
 func (c *Tctx) Delete(key string) error {
-	storeKey := c.prefix + "/" + key
-	_, e := store.Store.Delete(context.TODO(), storeKey)
-	if e != nil {
-		c.log.WithFields(log.Fields{
-			"error": e,
-			"key":   storeKey,
-		}).Error("failed to delete key")
-		return e
+	storeKey := c.config.StorePrefix + key
```

Comment:
What about the cached values here? The next get would still get the value from the cache.

Reply:
I had overlooked this. I was supposed to remove Delete () altogther, after confirming that it wasn't being used anywhere.

Reply:
Just for the sake of completeness, I'd like to have delete as well.

Reply:
Done.